pkg/mfa/client.go

```diff
@@ -263,11 +263,20 @@ func (c *Client) VerifyFactor(
 	return VerifyChallenge(ctx, opts)
 }
 
+type VerificationResponseError struct {
+	Code    string
+	Message string
+}
+
+func (r *VerificationResponseError) Error() string {
+	return fmt.Sprintf("code: %s. message: %v", r.Code, r.Message)
```

Comment:
Adding more context to the message would be nice probably?
```suggestion
	return fmt.Sprintf("mfa verification failed: %s (code: %s)", err.Message, err.Code)
```